CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+The Changelog has moved!
+
+You can find it at the following URL:
+
+s3://dl.influxdata.com/platform/nightlies/master/CHANGELOG.md
```

Comment:
Can we expose this over http (or something that you can type into the Chrome address bar at least?)

Reply:
Also let's make it a markdown link please

Reply:
oh yes, we can, i'll fix

Reply:
Note that the link will be a 404 until the first changelog is uploaded successfully